pages/learn/develop/local-mode.md

```diff
@@ -122,14 +122,14 @@ These containers will have access to all the features and environment that balen
 
 ## SSH into the running app container or host OS
 
-If we want to run some test commands in our app container, we can do this easily using `{{ $names.company.short }} local ssh`. This command drops us directly into the selected container:
+If we want to run some test commands in our app container, we can do this easily using `{{ $names.company.short }} ssh`. This command drops us directly into the hostOS. From here, we can check system logs and [perform other troubleshooting tasks][troubleshooting]:
```

Comment:
Suggestions: (feel free to disagree)
hostOS → host OS
From here → From there

Reply:
I'll do you one better and replace all `hostOS` with `host OS` (they won).